apps/emqx_auth_mysql/src/emqx_authn_mysql.erl

```diff
@@ -37,9 +38,11 @@ create(_AuthenticatorID, Config) ->
     create(Config).
 
 create(Config0) ->
-    ResourceId = emqx_authn_utils:make_resource_id(?MODULE),
+    ResourceId = emqx_authn_utils:make_resource_id(<<"mysql">>),
     {Config, State} = parse_config(Config0),
-    {ok, _Data} = emqx_authn_utils:create_resource(ResourceId, emqx_mysql, Config),
+    {ok, _Data} = emqx_authn_utils:create_resource(
+        ResourceId, emqx_mysql, Config, ?AUTHN_MECHANISM_BIN, ?AUTHN_BACKEND_BIN
+    ),
```

Comment:
nit: Notice that some function calls use hard coded atom such as `'mysql'`, and some use hard coded binary `<<"mysql">>`.
Using macros uniformly may help improve readability.

funcs:
call `emqx_authn_utils:make_resource_id/1` used binary
call `emqx_authz_utils:make_resource_id/1` used atom
but they all return `binary()`

call `emqx_authn_utils:create_resource/5` used binary as `owner_id`
call `emqx_authz_utils:create_resource/5` used atom as `owner_id`
And record field `owner_id` is new added, Unified with atom/binary might be better?

Reply:
changed to use macros everywhere (except for jwt).
it's ok to be `binary()` or `atom()` since it's only intended for logging.